src/main/java/org/mandas/docker/client/messages/HostConfig.java

```diff
@@ -459,6 +463,13 @@ default Builder hostPidMode() {
       return this;
     }
 
+    Builder usernsMode(String usernsMode);
+
+    default Builder hostUserNsMode() {
```

Comment:
I 'd suggest you to remove the `hostUserNsMode()`. It's not that it's incorrect, but in terms of symmetry I am trying to avoid convenience methods like this.

Thanks for the PR btw

Reply:
sure, removed